src/main/resources/doc/en/contents.xml

```diff
@@ -173,15 +173,15 @@
 			<tocitem target="io_joystick" text="Joystick" image="icon_joystick" />
 			<tocitem target="io_keyboard" text="Keyboard" image="icon_keyboard" />
 			<tocitem target="io_led" text="LED" image="icon_led" />
-			<tocitem target="io_ledRVB" text="LED RVB" image="icon_rvbled" />
+			<tocitem target="io_ledRVB" text="LED RGB" image="icon_rvbled" />
```

Comment:
Could you please also fix the typo in the `image` attribute? This might also require renaming `icon_rvbled` in other places.